src/action/index.js

```diff
@@ -27,6 +28,7 @@ when(
 when(
   () => store.walletReady,
   async () => {
+    await backup.authenticate();
```

Comment:
Look like we can remove this since we already authenticate above?

```suggestion
```

Reply:
Yeah good spot!